### PR TITLE
Adapt performance tests script for Strimzi-ci

### PR DESCRIPTION
**Pull request by franvila**
### Type of change

- Bugfix

### Description

Part of #1047, it is needed to adapt the script to be able to run the tests using strimzi-ci.

### Additional Context

Most important changes:
* Added `-pl kroxylicious-systemtests` param when getting the kafka version to get the proper version for the tests
* Added `PRODUCER_PROPERTIES` variable in case we want to add more properties in the future
* Added `KAFKA_IMAGE` into the script, because having only in the `docker_compose` file, the `KAFKA_VERSION` is not replaced properly when running on strimzi-ci.

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Review performance test results. Ensure that any degradations to performance numbers are understood and justified.
- [ ] Make sure all Sonarcloud warnings are addressed or are justifiably ignored.
- [ ] Update documentation
- [ ] Reference relevant issue(s) and close them after merging
- [ ] For user facing changes, update CHANGELOG.md (remember to include changes affecting the API of the test artefacts too).

### CODE REVIEW

**Comment by franvila** [comment]
@strimzi-ci run perf

File: performance-tests/perf-tests.sh [hunk 1]
```diff
@@ -21,11 +22,12 @@ NOCOLOR='\033[0m'
 
 KROXYLICIOUS_CHECKOUT=${KROXYLICIOUS_CHECKOUT:-${PERF_TESTS_DIR}/..}
 
-KAFKA_VERSION=${KAFKA_VERSION:-$(mvn -f ${KROXYLICIOUS_CHECKOUT}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=kafka.version -q -DforceStdout)}
+KAFKA_VERSION=${KAFKA_VERSION:-$(mvn -f ${KROXYLICIOUS_CHECKOUT}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=kafka.version -q -DforceStdout -pl kroxylicious-systemtests)}
 STRIMZI_VERSION=${STRIMZI_VERSION:-$(mvn -f ${KROXYLICIOUS_CHECKOUT}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=strimzi.version -q -DforceStdout)}
 KAFKA_TOOL_IMAGE=${KAFKA_TOOL_IMAGE:-quay.io/strimzi/kafka:${STRIMZI_VERSION}-kafka-${KAFKA_VERSION}}
+KAFKA_IMAGE=${KAFKA_IMAGE:-"quay.io/ogunalp/kafka-native:latest-kafka-${KAFKA_VERSION}"}
```
Comment (robobario) [inline performance-tests/perf-tests.sh:28]: Did you move this here just to have it all in one place and not split the defaulting over the shell script and compose file? Or is there some other reason?

**Review by robobario — APPROVED** [review]
LGTM

File: performance-tests/perf-tests.sh (hunk 1 above)
Comment (franvila) [reply performance-tests/perf-tests.sh:28]: For some reason, when running the `perf-tests.sh` script, the replacement of `KAFKA_VERSION` was not done. I mean, the default value of `KAFKA_IMAGE` remained `quay.io/ogunalp/kafka-native:latest-kafka-${KAFKA_VERSION}`, but setting it in the script, everything works fine.

**Pull request merged**